crates/compiler/fmt/src/pattern.rs

```diff
@@ -194,9 +228,22 @@ impl<'a> Formattable for Pattern<'a> {
 
                 buf.push_str("]");
             }
-            ListRest => {
+            ListRest(opt_pattern_as) => {
                 buf.indent(indent);
                 buf.push_str("..");
+
+                if let Some((list_rest_spaces, pattern_as)) = opt_pattern_as {
+                    // these spaces "belong" to the `..`, which can never be multiline
+                    fmt_comments_only(buf, list_rest_spaces.iter(), NewlineAt::Bottom, indent);
```

Comment:
is this sound reasoning?

Reply:
seems reasonable to me

Reply:
This smells like something that might cause failures in fmt fuzzing - but we can handle that later.